modules/rpc/thriftrw-plugin-thriftsync/README.md

```diff
@@ -3,17 +3,44 @@
  *.thrift file. With the use of thriftsync plugin, a user who needs to build a service should be able
  to auto generate the code and write service specific logic without worrying about underlying platform.
 
+## Usage
+Run `thriftsync` on any thrift file update to sync your handler code with the methods in the thrift file.
```

Comment:
- thrift --> Thrift (throughout)
- Meaning is unclear. Run the command on a thrift file…to sync? Should we cut "update"? I think we're either missing words or we have too many.